source/includes/_endpoint-user.md

```diff
@@ -76,4 +106,64 @@ The new owner must also be part of the same account and should have
 }
 ````
 
-The new owner provided can be a user URL or a user email.
\ No newline at end of file
+The new owner provided can be a user URL or a user email.
+##### User Datasets
+
+`/account/users/{id}/datasets/`
+
+This URL is only accessible and available to account admins. 
+
+This Shoji catalog lists all the datasets that are owned by this user.
+
+
+##### User Visible datasets
+
+`/users/{id}/visible_datasets/`
```

Comment:
Don't worry about it; we're going to be revising this catalog/authz layout anyway.